.github/workflows/publish-chart.yml

```diff
@@ -3,9 +3,6 @@ name: Publish main helm chart
 on:
   workflow_dispatch:
   push:
-    paths:
-      - 'helm/**'
-      - '.github/workflows/**'
```

Comment:
We need to make sure we use the latest Helm chart with the latest image. This is the simplest solution. We have pruning on GCS bucket anyway.